### PR TITLE
AOM-130: Proofreading and editing content on Add-on Manager

### PR DESCRIPTION
**Pull request by Charpell**
## JIRA TICKET NAME
[AOM-130: Proofreading and editing content on Add-on Manager](https://issues.openmrs.org/browse/AOM-130)

### SUMMARY
As we approach the final stage of releasing the Add-on manager, there are still minor important keys contents that needs to be changed.

This PR address this issue

### CODE REVIEW

**Review by segunolalive — COMMENTED** [review]
Looks good to me

**Comment by patlub** [comment]
Looks good.

File: tests/components/SingleAddon.test.js [hunk 1]
```diff
@@ -95,40 +95,28 @@ describe('<SingleAddon/>', () => {
...
+  it('Should render 4 td tags', () => {
+    expect(addonComponent.find("td")).to.have.length(4);
   });
 
   it('Should render 3 div tags', () => {
     expect(addonComponent.find("div")).to.have.length(3);
   });
 
-  it('Should render an image tag', () => {
-    expect(addonComponent.find("img")).to.have.length(1);
```
Comment (dkayiwa) [inline tests/components/SingleAddon.test.js:107]: Did this change require removing these tests?

File: app/js/components/manageApps/SingleAddon.jsx [hunk 2]
```diff
@@ -66,16 +66,6 @@ export default class SingleAddon extends React.Component {
 
     return (
       <tr key={key}>
-        <td>
-          <img
-            src={app.appDetails &&
-              app.appDetails.icon !== null &&
```
Comment (dkayiwa) [inline app/js/components/manageApps/SingleAddon.jsx:72]: What is the reason behind removing this?

File: tests/components/SingleAddon.test.js (hunk 1 above)
Comment (Charpell) [reply tests/components/SingleAddon.test.js:107]: Yes it does. I got rid of the first column on the addon manager. That is the logo column.

File: app/js/components/manageApps/SingleAddon.jsx (hunk 2 above)
Comment (Charpell) [reply app/js/components/manageApps/SingleAddon.jsx:72]: You said i should remove the first column on the addon manager. That is the logo column.

**Pull request merged**